src/app/shared/post-it-collections/post-it-collections.component.ts

```diff
@@ -184,15 +176,15 @@ export class PostItCollectionsComponent implements OnInit, OnDestroy {
   public createPostIt(postIt: PostItCollectionModel): void {
     postIt.initializing = true;
 
-    this.collectionService.createCollection(postIt.collection)
+    this.collectionService.createCollection(CollectionConverter.toDto(postIt.collection))
```

Comment:
We must not use services in the components anymore! What about dispatching an action which creates a collection? This applies to the other parts of this component as well. Or was this just a quick fix and you plan to work on it in the next pull-request?